### PR TITLE
Add support for reverse links to codegen

### PR DESCRIPTION
**Pull request by SferaDev**
Part 1 for support of revlinks, part 2 will be type inference in the SDK

### CODE REVIEW

File: .changeset/many-peaches-reflect.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'@xata.io/codegen': minor
+'@xata.io/client': minor
+'@xata.io/cli': minor
+---
+
+Add support for reverse links
```
Comment (SferaDev) [inline .changeset/many-peaches-reflect.md:7]:
```suggestion
Add support for reverse links to codegen
```

File: .changeset/many-peaches-reflect.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+'@xata.io/codegen': minor
+'@xata.io/client': minor
```
Comment (SferaDev) [inline .changeset/many-peaches-reflect.md:3]:
```suggestion
```

File: packages/client/src/schema/inference.ts [hunk 3]
```diff
@@ -14,6 +14,10 @@ export type BaseSchema = {
     | { name: string; type: 'link'; link: { table: string } }
     | { name: string; type: 'object'; columns: { name: string; type: string }[] }
   )[];
+  revLinks?: readonly {
+    table: string;
+    column: string;
+  }[];
```
Comment (SferaDev) [inline packages/client/src/schema/inference.ts:20]:
```suggestion
```

**Review by richardgill — APPROVED** [review]
LGMT

**Pull request merged**